### PR TITLE
#143 remove decrypt token

### PR DESCRIPTION
**Pull request by vlad-tokarev**

### CODE REVIEW

File: helms/odahu-flow-opa/policies/permissions.rego [hunk 1]
```diff
@@ -8,14 +8,22 @@ permissions := {
...
   roles.viewer : [
-      ["GET", ".*"]
+    	["GET", "api/v1/model/deployment.*"],
+    	["GET", "api/v1/model/packaging.*"],
+    	["GET", "api/v1/model/training.*"],
+    	["GET", "api/v1/configuration.*"],
+    	["GET", "api/v1/connection/*[a-zA-Z0-9-]*/*$"],
+    	["GET", "api/v1/packaging/integration.*"],
+    	["GET", "api/v1/toolchain/integration.*"],
+    	["GET", "health"]
```
Comment (mcdoker18) [inline helms/odahu-flow-opa/policies/permissions.rego]: Can you check the following URL returns 403 for a user with viewer role: `/api/v1/connection/health/decrypted`?
If it doesn't, then maybe we should mark the start and of a line explicitly in every URL regex?

**Review by mcdoker18 — CHANGES_REQUESTED** [review]

File: helms/odahu-flow-opa/policies/permissions.rego (hunk 1 above)
Comment (vlad-tokarev) [reply helms/odahu-flow-opa/policies/permissions.rego]: yes, returns.

File: helms/odahu-flow-opa/policies/permissions.rego [hunk 2]
```diff
@@ -8,14 +8,22 @@ permissions := {
     	[".*", "api/v1/model/packaging.*"],
     	[".*", "api/v1/model/training.*"],
     	["GET", "api/v1/configuration.*"],
-    	["GET", "api/v1/connection.*"],
+    	["GET", "api/v1/connection/*[a-zA-Z0-9-]*/*$"],
```
Comment (maxkochubey) [inline helms/odahu-flow-opa/policies/permissions.rego]: Should this regex match strings like "api/v1/connection/s0m3-CoNnectioN-8/" (with optional "/" ending)?

**Review by maxkochubey — APPROVED** [review]

**Review by mcdoker18 — APPROVED** [review]

File: helms/odahu-flow-opa/policies/permissions.rego (hunk 2 above)
Comment (vlad-tokarev) [reply helms/odahu-flow-opa/policies/permissions.rego]: yes

**Pull request merged**